frontend/src/index.tsx

```diff
@@ -56,11 +56,19 @@ const authMiddleware = new ApolloLink((operation, forward) => {
   return forward(operation);
 });
 
-const logoutLink = onError(({ networkError, graphQLErrors }) => {
+const logoutLink = onError(({ networkError, graphQLErrors }: ErrorResponse) => {
   if (networkError && process.env.REACT_APP_BASE_URL) {
-    console.error("network error", networkError);
-    console.log("redirecting to", process.env.REACT_APP_BASE_URL);
-    window.location.replace(process.env.REACT_APP_BASE_URL);
+    if ("statusCode" in networkError && networkError.statusCode === 401) {
+      console.warn("[UNATHORIZED_ACCESS] !!");
+      console.warn("redirect-to:", process.env.REACT_APP_BASE_URL);
+      window.location.replace(process.env.REACT_APP_BASE_URL);
+    } else {
+      showError(
```

Comment:
Can we add a log statement here as well
```suggestion
     appInsights.trackException({error: networkError});
      showError(
```